src/certificates/hooks/useCertificates.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { RequestStatus } from '../../data/constants';
+import getPageHeadTitle from '../../generic/utils';
+import {
+  getComponentMode, getLoadingStatus, getCertificates, getHasCertificateModes, getCourseTitle,
+} from '../data/selectors';
+import { setMode } from '../data/slice';
+import { fetchCertificates } from '../data/thunks';
+import { MODE_STATES } from '../data/constants';
+
+import messages from '../messages';
+
+const useCertificates = ({ courseId }) => {
+  const dispatch = useDispatch();
+  const intl = useIntl();
+
+  const componentMode = useSelector(getComponentMode);
+  const certificates = useSelector(getCertificates);
+  const loadingStatus = useSelector(getLoadingStatus);
+  const hasCertificateModes = useSelector(getHasCertificateModes);
+  const courseTitle = useSelector(getCourseTitle);
+
+  const isLoading = useMemo(() => loadingStatus === RequestStatus.IN_PROGRESS, [loadingStatus]);
+
+  useEffect(() => {
+    if (!hasCertificateModes) {
+      dispatch(setMode(MODE_STATES.noModes));
+    } else if (certificates.length === 0) {
+      dispatch(setMode(MODE_STATES.noCertificates));
+    } else {
+      dispatch(setMode(MODE_STATES.view));
+    }
+  }, [hasCertificateModes, certificates]);
+
+  useEffect(() => {
+    dispatch(fetchCertificates(courseId));
+  }, [courseId]);
+
+  document.title = getPageHeadTitle(courseTitle, intl.formatMessage(messages.headingTitleTabText));
```

Comment:
Remove this and use `<Helmet />` in `Certificates.jsx`. See `AccessibilityPages.jsx` [L20-L26](https://github.com/openedx/frontend-app-course-authoring/blob/master/src/accessibility-page/AccessibilityPage.jsx#L20-L26). `getPageHeadTitle` can still be used to format the mesage.